masterlist.yaml

```
@@ -5552,6 +5553,21 @@ plugins:
         crc: 0x51ED5F6A
         util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
         nav: 10
+      - <<: *patch3rdParty
```

It's missing the _key_ **msg** for messages.
```suggestion
    msg:
      - <<: *patch3rdParty
```